metrics-operator/converter/slo_converter.md

```diff
@@ -0,0 +1,303 @@
+# SLO Converter
+
+## Description
+
+SLO converter is a tool to convert `slo.yaml` files used in [KeptnV1](https://v1.keptn.sh/) into the new
+`AnalysisDefinition` resources used in the new kubernetes-native [Keptn](https://lifecycle.keptn.sh/).
+The converter is part of `metrics-operator` image.
+
+## Usage
+
+The converter will convert a single `slo.yaml` file into single `AnalysisDefintion` resource.
+
+To run the converter, execute the following command:
+
+```shell
+docker-run <METRICS_OPERATOR_IMAGE> manager --convert-slo=<PATH_TO_SLO> --slo-namespace=<ANALYSIS_VALUE_TEMPLATE_NAMESPACE> --definition=<DEFINITION_NAME>
+```
+
+Please be aware, you need to substitute the placeholders with the following information:
+
+* **PATH_TO_SLO** - path to your `slo.yaml` file
+* **ANALYSIS_VALUE_TEMPLATE_NAMESPACE** - namespace of `AnalysisValueTemplate` which will be referenced in objectives
+* **DEFINITION_NAME** - name of created `AnalysisDefinition`
+
+> **Note**
+
+All the SLOs present in `slo.yaml` file will reference `AnalysisValueTemplate` resources from the namespace defined
+by `ANALYSIS_VALUE_TEMPLATE_NAMESPACE` argument.
```

Comment:
will the AnalysisDefinitions then always be created in the same namespace as the AnalysisValueTemplates?

Reply:
the analysisdefinition does not have it's namespace defined, the namespace is defined only for the referenced `AnalysisValueTemplates`. The `AnalysisDefinition` itself can be applied to any namespace